packages/angular_devkit/build_angular/src/builders/application/index.ts

```diff
@@ -43,20 +43,12 @@ export async function* buildApplicationInternal(
 
   const normalizedOptions = await normalizeOptions(context, projectName, options);
 
-  // Warn about prerender/ssr not yet supporting localize
-  if (
-    normalizedOptions.i18nOptions.shouldInline &&
-    (normalizedOptions.prerenderOptions ||
-      normalizedOptions.ssrOptions ||
-      normalizedOptions.appShellOptions)
-  ) {
+  // Warn about SSR not yet supporting localize
```

Comment:
I am not fully understanding why this is not supported. To prerender, the i189n files are already inlined. Thus, this should also work no?

Reply:
The inlining should work but wanted to wait for the application builder output path updates first to avoid making any incorrect assumptions in the inlining code.